src/translations/sqlb_ar_SA.ts

```diff
@@ -2743,7 +2824,7 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
     <message>
         <location filename="../MainWindow.ui" line="1157"/>
         <source>Ctrl+O</source>
-        <translation></translation>
+        <translation>Ctrl+O</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.